### PR TITLE
Share wallet address [17865]

### LINKED ISSUE #17865: Share Screen -> Implement Share Wallet Address
From the designs:
https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=2937-643562&mode=design&t=r1UAixLCMYOEphsl-4

- Please align with @ulisesmac for the correct components to use for this as some adjustments were recently made for the QR sharing components.

Check with design of what account is to be shared here - is it the default wallet account or like the last viewed account? 

Acceptance Criteria:
- implement the UI and use real data where possible.
- Component spec test covering the screen. (entry added in status_im2/core_spec.cljs)

### PR DESCRIPTION
**Pull request by tumanov-alex**
fixes #17865 

## Share wallet address design update (UI only)
While I'm working on adding share-qr-code to the wallet share screen and re-using swipeable status component, I've decided to share the UI part asap

### Steps to test

- go to quo library
- scroll all the way down to `share-qr-code`

#### After:

| [Figma (if available) ](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=451-18993&mode=design&t=7fJFburgVjMSvySD-0)

status: ready

### CODE REVIEW

File: src/quo/components/share/share_qr_code/style.cljs [hunk 1]
```diff
@@ -60,26 +79,14 @@
...
 ;;; Dashed line
-(def divider-container
-  {:height            8
-   :margin-horizontal 4
-   :justify-content   :center
-   :overflow          :hidden})
-
 (def ^:private padding-for-divider (+ padding-horizontal 4))
 (def ^:private dashed-line-width 2)
 (def ^:private dashed-line-space 4)
```
Comment (ulisesmac) [inline src/quo/components/share/share_qr_code/style.cljs]: If we are not going to use these styles anymore, we should remove them 👍

File: src/quo/components/share/share_qr_code/view.cljs [hunk 2]
```diff
@@ -2,67 +2,74 @@
...
+     {:style style/header-and-avatar-container}
+     [account-avatar/view
+      (assoc props
+             :size 32
+             :type :default)]
+     [text/text
+      {:size :heading-2
+       :weight :semi-bold
+       :style
+       (style/header-title component-width)
```
Comment (ulisesmac) [inline src/quo/components/share/share_qr_code/view.cljs]: They should be in the same line (46)

File: doc/debugging.md [hunk 3]
```diff
@@ -7,13 +7,13 @@
...
 ```bash
-$ yarn shadow-cljs run re-frisk-remote.core/start
+yarn shadow-cljs run re-frisk-remote.core/start
 ```
 
 or you can also use make:
 
 ```bash
-$ make run-re-frisk
+make run-re-frisk
```
Comment (ulisesmac) [inline doc/debugging.md:16]: Maybe unrelated changes?

**Review by ulisesmac — COMMENTED** [review]
Please make sure to update the docstring and component API to properly reflect the new design.

Also, please update the preview screen and test if needed.

In PR's description, the screenshot of the new design doesn't show the configuration icon, so make sure it's being properly rendered.

File: doc/debugging.md (hunk 3 above)
Comment (tumanov-alex) [reply doc/debugging.md:16]: i know that i should create a separate pr for these two lines, but since can we smuggle them here? :)

**Review by OmarBasem — APPROVED** [review]

File: doc/debugging.md (hunk 3 above)
Comment (ulisesmac) [reply doc/debugging.md:16]: do we have an isue for it or something?

**Comment by ulisesmac** [comment]
Hey Alex, by loking at the screenshots seems the grandient is also missing :sweat_smile:
